Public/Src/Utilities/UnitTests/Executables/MockVmCommandProxy/Program.cs

```diff
@@ -102,7 +102,6 @@ private static int StartBuild(string inputFile)
 
             Console.WriteLine($"Start build request: ");
             Console.WriteLine($"    - HostLowPrivilegeUsername: {request.HostLowPrivilegeUsername ?? string.Empty}");
-            Console.WriteLine($"    - HostLowPrivilegePassword: {request.HostLowPrivilegePassword ?? string.Empty}");
 

```

Comment:
Why are we removing this? It is good to have for debugging. This class is only a mock.